spec/services/join_course_spec.rb

```diff
@@ -23,6 +23,34 @@
     enroll_as_instructor
   end
 
+  context 'with real name' do    
+    let(:course) { create(:basic_course) }
```

Comment:
I wasn't quite sure which course you wanted to test this on, but I assumed a `basic_course` would work find. Let me know.

Reply:
Testing it with `create(:course)` is the default pattern, but this is fine as well.

Reply:
I can change it if needed.

Reply:
no need. :-)